### PR TITLE
ft(BKTCLT-13): Add exported types

### PR DESCRIPTION
**Pull request by tmacro**

### CODE REVIEW

File: index.d.ts [hunk 1]
```diff
@@ -0,0 +1,36 @@
...
+
+        getPort(): number;
+        getRaftInformation(bucketName: string, reqUids: string[], callback: Callback, reqLogger?: any): void;
+        getBucketLeader(bucketName: string, reqUids: string[], callback: Callback, reqLogger?: any): void;
+        getBucketAttributes(bucketName: string, reqUids: string[], callback: Callback, reqLogger?: any): void;
+        putBucketAttributes(bucketName: string, reqUids: string[], attributes: string, callback: Callback, reqLogger: any): void;
+        createBucket(bucketName: string, reqUids: string[], attributes: string, callback: Callback, reqLogger: any): void;
+        deleteBucket(bucketName: string, reqUids: string[], callback: Callback, reqLogger: any): void;
+        putObject(bucketName: string, objName: string, objVal: string, reqUids: string[], callback: Callback, params: any, reqLogger: any): void;
+        getObject(bucketName: string, objName: string, reqUids: string[], callback: Callback, params, reqLogger: any): void;
```
Comment (dvasilas) [inline index.d.ts]:
```suggestion
        getObject(bucketName: string, objName: string, reqUids: string[], callback: Callback, params: any, reqLogger: any): void;
```

**Review by jonathan-gramain — COMMENTED** [review]
We should retarget this PR to either `development/7.10` or create `development/7.70`, as `development/8` is reserved for Artesca.

If changes are only typescript-related, I think `development/7.10` is a good target to avoid future conflicts. For more involved/risky changes, we would have to create the 7.70 branch.

**Comment by jonathan-gramain** [comment]
On the other hand I'm not against merging S3C and artesca in a single branch, but it should be consciously decided on.

File: index.d.ts (hunk 1 above)
Comment (dvasilas) [reply index.d.ts]: Same comment after the branch change. Maybe a missing commit.

**Review by dvasilas — APPROVED** [review]

**Review by jonathan-gramain — APPROVED** [review]

File: index.d.ts (hunk 1 above)
Comment (tmacro) [reply index.d.ts]: Seems to have been lost in the rebase.

**Pull request merged**